### PR TITLE
feat: [BREAKING] remove adapter from Action class

### PR DESCRIPTION
**Pull request by fsher**

### CODE REVIEW

**Review by prsvic — APPROVED** [review]
Like👍

File: packages/blinks-core/src/api/Action/action-supportability.ts [hunk 1]
```diff
@@ -44,18 +45,21 @@ export type ActionSupportability =
 
 export type ActionSupportStrategy = (
   action: Action,
+  adapter?: ActionAdapter,
```
Comment (tsmbl) [inline packages/blinks-core/src/api/Action/action-supportability.ts:48]: I think we changing the behaviour when keep it optional here

Can you please doublecheck that e.g. monad blink shows incompatible fallback? 

solana-action:https://test-action-inputs.dialect.workers.dev/monad-poc/transfer

**Review by tsmbl — CHANGES_REQUESTED** [review]

File: packages/blinks-core/src/api/Action/action-supportability.ts (hunk 1 above)
Comment (fsher) [reply packages/blinks-core/src/api/Action/action-supportability.ts:48]: will test, thank you!

**Review by tsmbl — APPROVED** [review]
Lgtm

**Pull request merged**